modules/test/tls/python/requirements.txt

```diff
@@ -1,2 +1,4 @@
 cryptography
-pyOpenSSL
\ No newline at end of file
+pyOpenSSL
+pyshark
+requests
```

Comment:
Need to pin requirements for tls module.  Package updates can cause python crash if they get out of sync.
https://github.com/google/testrun/pull/366